unit-tests/unit-test-config.py

```diff
@@ -90,7 +90,10 @@ def generate_cmake( builddir, testdir, testname, filelist ):
 set( SRC_FILES ''' + filelist + '''
 )
 add_executable( ''' + testname + ''' ${SRC_FILES} )
-source_group( "Common Files" FILES ${ELPP_FILES} ${CATCH_FILES} )
+source_group( "Common Files" FILES ${ELPP_FILES} ${CATCH_FILES}''' )
+    if not custom_main:
+        handle.write( ' ' + dir + '/unit-test-default-main.cpp' )
```

Comment:
So `unit-test-default-main.cpp` is in common files and in source files. Is that necessary?